varcode/core_logic.py

```diff
@@ -381,6 +383,78 @@ def infer_exonic_effect(variant, transcript):
         variant,
         transcript)
 
+def find_nearest_exon(start, end, exons):
+    """
+    Finds nearest exon to an intronic variant, returns the
+    distance to that exon, along with the Exon object itself.
+    """
+    best_distance = float("inf")
+    best_exon = None
+    for exon in exons:
+        if exon.start > end:
+            # distance from end of variant to start of exon
+            # (when variant starts before the exon)
+            distance = exon.start - end
+        elif exon.end < start:
+            distance = start - exon.end
+        else:
+            assert exon.start <= end and exon.end >= start, \
```

Comment:
This is more of a documenting assert, to remind the reader what the else branch means.